lib/CMakeLists.txt

```
@@ -9,4 +9,6 @@ if (${AIEBU_AIE_RT_BIN_DIR} STREQUAL ${AIEBU_BINARY_DIR})
   aiebu_add_subdirectory_disable_install_target(aie-rt/driver)
 endif()
 
-add_subdirectory(src)
+if (${AIEBU_NATIVE_BUILD})
```

```suggestion
if (AIEBU_NATIVE_BUILD)
```

If you write if(${FOO}), CMake first expands ${FOO} before evaluating the condition. This can lead to subtle bugs, especially if FOO expands to a value that is itself the name of another variable or an empty string